### DISCUSSION TITLE
Fortnite Questions

### DISCUSSION THREAD

**Discussion by Grayson9352** [guttir14/UnrealDumper-4.25#15] [Q&A]
Is it updated for 16.10 and how could i possibly do it for 6.30. I kind of know but like i'm not too sure on it. Also this only dumps offsets right?

**Comment by guttir14** [comment]
My discord is @Googloqe#4257
I wrote dumper for old Fortnite not so long ago. I can send it to you.
This dumper dumps definitions of structures (so, you get offsets to fields, sizes of structures, functions of these structures). Also it dumps objects and names arrays.
